src/main/kotlin/com/terraformation/backend/customer/db/ProjectStore.kt

```diff
@@ -47,17 +50,32 @@ class ProjectStore(
   }
 
   fun fetchCohortData(projectId: ProjectId): ProjectCohortData? {
+    requirePermissions { readProject(projectId) }
+
     return dslContext
```

Comment:
Nice query! This may be a valuable pattern for the ModuleStore/DeliverableStore as well so applications can use the same endpoints to fetch "non-cohort" modules.